docs/fsharp/language-reference/parameters-and-arguments.md

```diff
@@ -134,6 +134,15 @@ type C =
         printfn "%s" message
 ```
 
+The value given as argument to `DefaultParameterValue` must match the type of the parameter, i.e. the following is not allowed:
+
+```fsharp
+type C() =
+    static member Wrong([<Optional;DefaultParameterValue("string")>] i:int) = ()
+```
+
+In this case compiler generate a warning, and ignore both attributes altogether. Note that the default value `null` needs to be type-annotated, as otherwise the compiler infers the wrong type, i.e. `[<Optional;DefaultParameterValue(null:obj)>] o:obj`.
```

Comment:
Change to: "In this case, the compiler generates a warning and will ignore both attributes altogether." or some variant, as the current first sentence is not grammatically correct.

Reply:
My bad.